go/mysql/connection.go

```diff
@@ -57,5 +94,9 @@ func (this *ConnectionConfig) GetDBUri(databaseName string) string {
 		hostname = fmt.Sprintf("[%s]", hostname)
 	}
 	interpolateParams := true
-	return fmt.Sprintf("%s:%s@tcp(%s:%d)/%s?interpolateParams=%t&autocommit=true&charset=utf8mb4,utf8,latin1", this.User, this.Password, hostname, this.Key.Port, databaseName, interpolateParams)
+	tlsOption := "false"
```

Comment:
It might be worth leaving a comment that "false" is defaulted, so your explicit use of it is for code path simplicity. Up to you.